### ISSUE TITLE
[Bug]: Secure folder rests password each time opened

### ISSUE THREAD

**Issue by loststone2** [LeeXDA21/UnboundHub#7]
### What is your Telegram Username?

@Loststone2

### Device

SM-G985F (Galaxy S20+ 4G)

### Unbound Version

V1.3.0

### Bug Description

Secure Folder password setup is prompted each time SF is opened. 
Video of issue and what I did while logging https://linksharing.samsungcloud.com/edle3vogs6Po

### Bug Visual Description

Secure Folder password setup is prompted each time SF is opened.

https://user-images.githubusercontent.com/19869342/184380361-3cd82665-261d-4fc7-9955-afbade0445e1.mp4

![Screenshot_20220812-103006_Settings](https://user-images.githubusercontent.com/19869342/184380375-9e9f5f48-b8c2-4da6-840a-3de539a50ae7.jpg)
![Screenshot_20220812-103018_Settings](https://user-images.githubusercontent.com/19869342/184380381-abb67e38-7f51-4c5f-b706-31f034b66bfb.jpg)
 

### Log File(s)

[log_e.log](https://github.com/LeeXDA21/UnboundHub/files/9320764/log_e.log)

https://drive.google.com/file/d/1xKkrRxqMag_LOSQqgUDoJ9fUbvVtQMDL/view?usp=sharing

Logged while opening Secure Folder and it prompted for a new password. Hopefully, this proves useful 🤞

### Additional checks

- [X] I have checked other bug reports and this is not a duplicate.
- [X] This is a bug in Project Unbound and NOT any other ROM.
- [X] You understand and declare that if any information provided here is incorrect or instructions have not been followed correctly that it can lead to you being permanently banned from all Unbound owned servers.

**Comment by LeeXDA21** [comment]
@loststone2 V1.4 is being produced. This will be looked into further.

**Comment by loststone2** [comment]
Awesome Lmk If there's any way I can help. Put more important details on the issue below

This is the VbMeta and TWRP I flashed. 
https://drive.google.com/file/d/1P3ybW_OfmIONKMpZ0te58G9YMpLJcihe/view?usp=sharing
https://drive.google.com/file/d/1N5X-Cv9riI7Cx6KQjydd64Z7-Rpei-f9/view?usp=sharing

I did a clean install and I have flashed the wrong VbMeta after ROM was installed  ( Causing the 32GB storage issue). Its should be considered the issue could have been caused due to human error flashing.
I can't recall if I reflashed the whole stock os after using the correct vbmeta file for reinstalling the ROM 

When V4 Comes out. I plan to do a clean install again. Let me know if there's any way I can help. 

Appreciate your work : )
